Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
-ruby File.read(File.join(__dir__, '.ruby-version')).strip
+ruby ">= 2.6"
```

Comment:
Have you tested with latest 3.2.1? Last I checked, CocoaPods just crashed.

Reply:
I tested with 3.1.2 and 3.1.3, actually.

Good call, let me check with those as well.

Reply:
Good call @tido64: cocoapods explodes because of this error:
```
NoMethodError - undefined method `=~' for []:Array
/Users/cipolleschi/.rbenv/versions/3.2.0/lib/ruby/gems/3.2.0/gems/cocoapods-1.11.3/lib/cocoapods/target/build_settings.rb:446:in `block (2 levels) in merged_xcconfigs'
```

Looking at the [reported line](https://github.com/CocoaPods/CocoaPods/blob/master/lib/cocoapods/target/build_settings.rb#L446) it seems that they are using =~.

However [this commit](https://github.com/CocoaPods/CocoaPods/commit/93c6c8a3a8199b27150634b015dfd0a0614347dc) seems like it has fixed it, but they haven't done a release of Cocoapods yet with the fix.

Reply:
![CleanShot 2023-02-24 at 16 19 06](https://user-images.githubusercontent.com/49578/221231528-baf1a2c2-4f0e-43db-b494-6646f114935c.png)

We should document that RVM doesn't relative versions (only absolute versions) when navigating into a project directory.